### DISCUSSION TITLE
Reply button missing

### DISCUSSION THREAD

**Discussion by AlisonMV** [insin/control-panel-for-twitter#193] [Q&A]
I'm not a programmer, but this is the only place I could find where I might get help. Overnight, I've lost the "reply" button in my Tweak New Twitter Chrome extension. Already been through all the extension options, and can't see anything that specifically addresses this, or get it back by turning the extension off and back on again, turning Twitter off and back on again, or restarting my laptop. What am I missing???

**Comment by CozmoB** [comment]
Same issue here
